src/main/java/seedu/intrack/model/internship/Internship.java

```diff
@@ -125,18 +123,18 @@ public boolean equals(Object other) {
         Internship otherInternship = (Internship) other;
         return otherInternship.getName().equals(getName())
                 && otherInternship.getPosition().equals(getPosition())
+                // && otherInternship.getStatus().equals(getStatus())
                 && otherInternship.getPhone().equals(getPhone())
                 && otherInternship.getEmail().equals(getEmail())
-                && otherInternship.getStatus().equals(getStatus())
                 && otherInternship.getAddress().equals(getAddress())
-                && otherInternship.getTasks().equals(getTasks())
+                // && otherInternship.getTasks().equals(getTasks())
```

Comment:
Would it be better to also check for equality in tasks?

Reply:
```suggestion
                && otherInternship.getTasks().equals(getTasks())
```